test/navigation/navigators/TrusteeRoutes.scala

```diff
@@ -173,11 +173,20 @@ trait TrusteeRoutes {
         }
       }
 
-      "go to TrusteeTelephoneNumberPage from TrusteeLivesInUKPage" in {
+      "go to TrusteesUkAddressPage from TrusteeLivesInUKPage" in {
         forAll(arbitrary[UserAnswers]) {
           userAnswers =>
 
```

Comment:
This should be constrained to only navigate if they have answered 'Yes' to living in the UK.

Reply:
Done.